assets/js/research.js

```diff
@@ -0,0 +1,133 @@
+/*Research Details Table*/
+
+const researchTable = document.querySelector(".main");
+const citeTable = document.querySelector(".citeT");
+
+const research = [
+    {
+        title : "A Batch Normalized Inference Network Keeps the KL Vanishing Away",
+        authors : "Qile Zhu, Wei Bi, Xiaojiang Liu, Xiyao Ma, Xiaolin Li and Dapeng Wu",
+        conferences : "The 58th Annual Meeting of the Association for Computational Linguistics",
+        year : ", 2020",
+        researchYr : "2 Jan, 2020",
```

Comment:
This is not required. Only year is sufficient

Reply:
Okay sir